fix (DPU-agent): make BgpPeeringRouteServer redundancy aware (#4708)
NVIDIA/bare-metal-manager-core: docs/architecture/health/health_probe_ids.md

### `BgpPeeringRouteServer`

> Indicates that a BGP session with a route server that is part of the NICo control plane could not be established by a host/DPU. Route servers are redundant, so this alert only applies the `PreventAllocations` and `PreventHostStateChanges` classifications when there is one healthy session left or none (including a single configured route server going down). with more than one healthy session, it's just a warning.

Author: I just capitalized it and fixed  the issue

Indicates that a BGP session with a route server that is part of the NICo control plane could not be established by a host/DPU. Route servers are redundant, so this alert only applies the `PreventAllocations` and `PreventHostStateChanges` classifications when there is one healthy session left or none (including a single configured route server going down). With more than one healthy session, it's just a warning.

> Indicates that a BGP session with a route server that is part of the NICo control plane could not be established by a host/DPU. Route servers are redundant, so this alert only applies the `PreventAllocations` and `PreventHostStateChanges` classifications when there is one healthy session left or none (including a single configured route server going down). With more than one healthy session, it's just a warning. A session is healthy when its state is `Established`; each configured route server that isn't gets its own alert.

Reviewer: ```suggestion
Indicates that a host/DPU could not establish a BGP session with a route server that is part of the NICo control plane.

Route servers are redundant. This alert only applies the `PreventAllocations` and `PreventHostStateChanges` classifications when there is _at most_ one healthy session left (including a single configured route server going down). With more than one healthy session, this is just a warning. A session is healthy when its state is `Established`; each configured route server that is not gets its own alert.
```

Author: Thanks for the review, I just fixed it and pushed it again.
